tests/core/data/test_sql_table_data_node.py

```diff
@@ -127,40 +132,27 @@ def test_read(self, mock_read_as, mock_read_as_pandas_dataframe, mock_read_as_nu
             properties=properties,
         )
 
-        assert sql_data_node_as_pandas.read() == "pandas"
+        assert sql_data_node_as_pandas._read() == "pandas"
```

Comment:
Why don't you use the read method anymore?
Maybe we need to test both methods, no?

Reply:
Hi, something went wrong with my git and pre-commit, I have changed it back to read(). Can you have a look again?

Reply:
Good to go!